pdns/pdns_recursor.cc

```diff
@@ -2566,6 +2567,9 @@ int serviceMain(int argc, char*argv[])
 
   showProductVersion();
   seedRandom(::arg()["entropy-source"]);
+  openssl_thread_setup();
+  openssl_seed();
```

Comment:
I don't believe we need the randomness anyway when only verifying signatures (although something about ecc is nagging in the back of my mind)